lib/shopify_api/event_pipe/event.ex

```diff
@@ -1,4 +1,17 @@
 defmodule ShopifyAPI.EventPipe.Event do
+  @derive {Jason.Encoder,
+           only: [
+             :destination,
+             :app,
+             :shop,
+             :token,
+             :object,
+             :callback,
+             :action,
+             :assigns,
+             :response
+           ]}
```

Comment:
Following in the examples of previous usage in this library, I've made the struct fields to be serialized explicit and opt-in, but it might make more sense for this struct in particular to just allow serializing all fields.

Reply:
Makes sense to me, feel free to simplify it down to that 👍

Reply:
Done - can always consider excluding new fields if they include particularly sensitive data.